skbio/io/format/blast6.py

```diff
@@ -0,0 +1,287 @@
+"""
+BLAST+6 format (:mod:`skbio.io.format.blast6`)
+=========================================================================
+
+.. currentmodule:: skbio.io.format.blast6
+
+The BLAST+6 format (``blast6``) stores strings, integers, and floats relating
```

Comment:
Can you explain that BLAST+6 format includes BLAST+ output format 6 and BLAST output format 8?